.github/workflows/performance-benchmarks.yml

```diff
@@ -4,7 +4,7 @@ on:
 
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest-8-cores
```

Comment:
original ticket mentions beefing up, but here we are downgrading arent we?

Reply:
That's correct.

Reply:
shouldnt we upgrade to 16?

Reply:
What is the rationale behind upgrading to 16 cores?

Reply:
Original ticket was tackled with PR: https://github.com/mattermost/mattermost-webapp/pull/12215 
Upgrading from 2 cores to 8 cores. 

This PR, as mentioned in the description, better sizes the performance benchmarks instances used for CI, downsizing from 8-core to 4-core.

Hope that makes sense @M-ZubairAhmed